package.json

```diff
@@ -106,12 +99,5 @@
     "url": "https://github.com/thirdweb-dev/typescript-sdk/issues"
   },
   "homepage": "https://github.com/thirdweb-dev/typescript-sdk#readme",
-  "author": "thirdweb",
```

Comment:
this was an extraneous leftover thing, all of this is defined in the tsup.config.ts file anyways

Reply:
isnt that what gets picked up by npm though? like to show on our npm page?